packages/cdk/lib/transcription-service.ts

```diff
@@ -619,15 +619,24 @@ export class TranscriptionService extends GuStack {
 			],
 		});
 
-		const volume = {
-			name: `${mediaDownloadApp}-volume`,
+		const downloadVolume = {
+			name: `${mediaDownloadApp}-download-volume`,
 		};
-		mediaDownloadTask.taskDefinition.addVolume(volume);
+		const tempVolume = {
```

Comment:
might be worth a comment explaining why this is needed

Reply:
good shout